pkg/constants/constants.go

```diff
@@ -114,6 +114,10 @@ const (
 	// SyncsetPauseAnnotation is a annotation used by clusterDeployment, if it's true, then we will disable syncing to a specific cluster
 	SyncsetPauseAnnotation = "hive.openshift.io/syncset-pause"
 
+	// HiveManagedAnnotation is added to any resources we sync to the remote cluster to help identify that they are
```

Comment:
nit: I feel 'HiveManagedAnnotation is an annotation added' will be more consistent